libs/platform/src/lib/components/panel/panel-content/panel-content.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
+
+@Component({
+    selector: 'fdp-panel-content',
+    templateUrl: './panel-content.component.html',
+    styleUrls: ['./panel-content.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PanelContentComponent {
+    /**
+     * @harmful Potentially bad approach to hardcode css related properties
+     * Custom height of the content container.
+     */
+    @Input()
+    contentHeight: string;
```

Comment:
I noticed the `@harmful` note. In the tech spec, we designed it so that app developer could apply CSS styles directly to the `fdp-panel-content` like this:
```html
<fdp-panel
    [id]="'panel-id'"
    [title]="'Panel Title'"
    [contentDensity]="'compact'"
    [collapsable]="true"
    [expanded]="true"
    (panelExpandChange)="onExpandChange($event)">
    
    <!-- future feature enhancement -->
    <fdp-panel-actions> 
        <fdp-button>Edit</fdp-button>
        <fdp-button>Delete</fdp-button>
    </fdp-panel-actions>
    
    <fdp-panel-content class="small-panel">
         Panel content goes here.
    </fdp-panel-conent>
</fdp-panel>

<!-- For fixed height panel -->
<style>
  .small-panel {
     height: 10rem;
  }
</style>
```
See https://app.gitbook.com/@fundamental-ngx/s/docs/platform-panel#example

Is this not possible given we're building this on a Core component? Would we need to change the Core code to accomplish this? Or can the app developer still apply CSS class (or styles directly) to size the panel content?

Reply:
I added this input cause it was requirement in the spec.
But assigning user's value directly to html as inline styles is considered as a bad practice, kind of hard code.
That's why we added this `@harmful` and showed in the example the way how user can restrict panel-content using simple style approach.

Reply:
OK. Sounds good.